HomeToGather/HomeToGather/View/Create/Last/ColorView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct ColorView: View {
+    @State var color: String
+    @Binding var selectedColor: String
+    
+    var body: some View {
+        ZStack {
+            
+            Circle()
+                .fill(Color.getColor(color: color))
+                .frame(width: 50, height: 50)
+            
+            if selectedColor == color {
```

Comment:
```suggestion
            if selectedColor is color {
```

`is`와 `==`는 어떻게 다를까요 ?  후후